starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -5,39 +5,47 @@
   <!-- Google fonts gives us access to the Open Sans Font -->
   <!-- Google fonts is also awesome! Make sure to check it out -->
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  French Onion Soup
-
-  We have been trying French onion soup in restaurants for years and my family
-and friends agree none can compare to my recipe for taste and simplicity of
-preparation.
-
-Ingredients
-1/2 cup unsalted butter
-2 tablespoons olive oil
-4 cups sliced onions
-4 (10.5 ounce) cans beef broth
-2 tablespoons dry sherry (optional)
-1 teaspoon dried thyme
-salt and pepper to taste
-4 slices French bread
-4 slices provolone cheese
-2 slices Swiss cheese, diced
-1/4 cup grated Parmesan cheese
-
-Directions
-Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
-continually stir until tender and translucent. Do not brown the onions.
-
-Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
-minutes.
-
-Heat the oven broiler.
-
-Ladle soup into oven safe serving bowls and place one slice of bread on top of each
-(bread may be broken into pieces if you prefer). Layer each slice of bread with a slice of
-provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan cheese. Place bowls on
-cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.
+  <div class="container">
+    <header>
+      <div class="title">
+        <h1>French Onion Soup</h1>
+      </div>
+      <div class="content">
+        <p>We have been trying French onion soup in restaurants for years and my family and friends agree none can compare to my recipe for taste and simplicity of preparation.</p>
+        <img src="images/recipe-info.png" alt="recipe info">
+      </div>
+    </header>
+    <hr>
+    <section>
+      <h2>Ingredients</h2>
+      <ul>
+        <li>1/2 cup unsalted butter</li>
+        <li>2 tablespoons olive oil</li>
+        <li>4 cups sliced onions</li>
+        <li>4 (10.5 ounce) cans beef broth</li>
+        <li>2 tablespoons dry sherry (optional)</li>
+        <li>1 teaspoon dried thyme</li>
+        <li>salt and pepper to taste</li>
+        <li>4 slices French bread</li>
+        <li>4 slices provolone cheese</li>
+        <li>2 slices Swiss cheese, diced</li>
+        <li>1/4 cup grated Parmesan cheese</li>
+      </ul>
+      <img src="images/cooking-info.png" alt="cooking information">
+    </section>
+    <hr>
+    <section>
+      <h2>Directions</h2>
+      <ol>
+        <li>Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and continually stir until tender and translucent. Do not brown the onions.</li>
```

Comment:
for this list instead of using ordered list you could have done `list-style: none` and just added 1) 2) 3) to the list items since it was only a few items.

pro way would be to do it like we did in class
`
ol {
counter-reset: custom;
list-style-type: none;
line-height: 1.6;
margin: 0 auto;
}

ol li:before {
content: counter(custom)') ';
counter-increment: custom;
}
`